docs/KarmaCommands.md

```diff
@@ -0,0 +1,60 @@
+# Karma Command Reference
+
+Command to giveth or taketh away a user's karma
+
+## Options
+
+### For normal users
+
+#### Add 1 karma to user
+
+```sh
+/rocket @user ++
```

Comment:
Personally I don't really like how this would be inconsistent with other commands, especially since we'd _also_ have a `/rocket karma` command still. See my other comment though - if we take away the ability to remove karma, we'd be able to have `/rocket karma @user`, which isn't much more to type than `/rocket @user ++`.